server/master/src/main/java/org/apache/accumulo/master/tableOps/TableInfo.java

```diff
@@ -32,6 +33,14 @@
   char timeType;
   String user;
 
+  // Record requested initial state at creation
+  TableCreationMode creationMode;
+
+  // Track information related to initial split creation
+  int initialSplitSize;
+  String splitFile;
+  String splitDirsFile;
+
   public Map<String,String> props;
 
   public String dir = null;
```

Comment:
Renaming this to defaultTabletDir would have helped me understand some of the code a bit faster.